src/main/java/Duke.java

```diff
@@ -1,10 +1,359 @@
+import Exceptions.DukeException;
+import Tasks.Deadline;
+import Tasks.Event;
+import Tasks.Task;
+import Tasks.Todo;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+import java.util.ArrayList;
+
 public class Duke {
-    public static void main(String[] args) {
+
+    //declare all constant integers
+    private static final int MAX_ARRAY_LENGTH = 100;
+
+    //declare all constant strings
+    private static final String lineLogo = "-----------------------------";
+    private static final String taskIncomplete = Character.toString((char) 0x2718);
+    private static final String taskComplete = Character.toString((char) 0x2713);
+
+    //declare string variables
+    private static String userInput = " ";
+
+    //declare array to store list of tasks
+    private static ArrayList<Task> taskList = new ArrayList<Task>();
+
+
+    //main method
```

Comment:
Consider removing these comments as well as the other comments below as they don't seem to contribute much to the readability of the code. You will be learning about JavaDoc comments which can be used instead.